packages/core/src/Stores/client-store.js

```diff
@@ -414,7 +414,6 @@ export default class ClientStore extends BaseStore {
                 this.is_pre_appstore = !!trading_hub;
                 localStorage.setItem('is_pre_appstore', !!trading_hub);
                 this.setPreferredLanguage(lang_from_url);
-                LocalStore.set(LANGUAGE_KEY, lang_from_url);
             }
```

Comment:
LocalStore.set(LANGUAGE_KEY, lang_from_url); is already set inside `this.setPreferredLanguage(lang_from_url);`
hence removing duplicate code

Reply:
<img width="636" alt="Screenshot 2023-03-01 at 8 24 32 AM" src="https://user-images.githubusercontent.com/122449658/222044603-9da02b7b-11f0-46e0-88fc-b63238c3acf0.png">
Could you please check whether the LocalStore function inside setPreferredLanguage accepts string value? Are both **LocalStore.set** and **LocalStore.setObject** similar in behaviour?
Last time when i was console logging line 417 with LocalStore.setObject, string values were not getting stored. it was expecting object type i believe. please check.

Reply:
@shaheer-deriv I checked the function. It is accepting string values and setting localstore values